Assets/MRTK/Providers/Oculus/XRSDK/Controllers/OculusXRSDKTouchController.cs

```diff
@@ -8,7 +8,7 @@
 using Unity.Profiling;
 using UnityEngine.XR;
 
-#if OCULUS_ENABLED
+#if OCULUSINTEGRATION_PRESENT
```

Comment:
This feels like it should still be `OCULUS_ENABLED`, right? This file doesn't depend on the Oculus integration, just the Oculus XR Plugin package, and this define is in the asmdef:

https://github.com/microsoft/MixedRealityToolkit-Unity/blob/258e3ebab93bd86a7aed6ef2517884e0a079caa2/Assets/MRTK/Providers/Oculus/XRSDK/Microsoft.MixedReality.Toolkit.Providers.XRSDK.Oculus.asmdef#L23-L29

Reply:
The problem is that the integration tool has no mechanism to remove/bring back the unity.xr.oculus package when the user tries to clean up the repository after removing oculus references. They has also been some CI issues regarding leaving a reference to Unity.XR.Oculus in the asmdef references, as by default the project doesn't import that package. Given that a large part of our focus for this project is to let users play with articulated hands tools with quest, I think it's alright to overall tie our Oculus story to the Integration Package asset.

Reply:
Does the integration tool add / remove this reference? I thought we shipped this as-is.

We should make sure we're on the same page with https://github.com/microsoft/MixedRealityToolkit-Unity/pull/8591 as well, which is looking to make sure we distribute properly 2019-ready asmdefs for UPM.

Reply:
Checked up with that PR and only one asmdef update should be redundant. We can merge in with whichever PR gets in first.